.github/ISSUE_TEMPLATE/releases/release_checklist.md

```diff
@@ -49,37 +49,34 @@
           - NuGet-client (except for feature band updates)
           - fsharp (except for feature band updates)
           - VSTest (except for feature band updates)
-1.  - [ ] [2.1] Fetch internal git data for submodules.
-      1.  Run `git submodule update --init --recursive` and see errors due to missing commits.
-      1.  Run `fetch-vsts-commits.sh` with a PAT to automatically fetch internal refs.
-      1.  Run `git submodule update --init --recursive` again to update failed submodules.
-          * If some refs are still missing on unusual repos, those repos may need to be set up in `fetch-vsts-commits.sh`.  Note: Repos in `fetch-vsts-commits.sh` don't need to be cleaned up.  
 1.  - [ ] Regenerate patches as necessary.
       - Run `./build.sh` to begin clone and patch application and see what, if anything, needs to be regenerated.  Issue to track re-patching workflow: [#1468](https://github.com/dotnet/source-build/issues/1468)
-      - [3.1] ASP.NET will always need to be regenerated because it contains a version number that needs to be updated.  The version should be updated to the SDK version being built.  See https://github.com/dotnet/source-build/blob/release/3.1/patches/aspnetcore/0007-Fix-version-number.patch
-1.  - [ ] [Internal] [3.1] Update internal feed list in `.\build.sh`
-      - Internal feeds are listed under `VSS_NUGET_EXTERNAL_FEED_ENDPOINTS`
-      - Make sure to keep `dotnet3.1-internal-transport` in the list
-      - Issue tracking this: [#1674](https://github.com/dotnet/source-build/issues/1674)
+      - [3.1] ASP.NET will always need to be regenerated because it contains a version number that needs to be updated.  The version should be updated to the SDK version being built.  See https://github.com/dotnet/source-build/blob/release/3.1/patches/aspnetcore/0006-Fix-version-number.patch
 1.  - [ ] Start building locally (for quick diagnosis) and in CI (for super clean build) and iterate towards a green build.
       - [Internal] [3.1] Set environment variable `internalPackageFeedPat` to dnceng internal PAT with package read and code read access.
```

Comment:
This should include the steps on how to retrieve the PAT.

Reply:
Added instructions for generating a PAT.